desc/compute/_basis_vectors.py

```diff
@@ -505,6 +505,29 @@ def _e_sup_theta(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="e^vartheta",
+    label="\\mathbf{e}^{\\vartheta}",
+    units="m^{-1}",
+    units_long="inverse meters",
+    description="Contravariant PEST poloidal basis vector",
+    dim=3,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["e^theta", "e^rho", "e^zeta", "theta_PEST_r", "theta_PEST_t", "theta_PEST_z"],
+    aliases=["e^theta_PEST"],
+)
+def _e_sup_theta_PEST(params, transforms, profiles, data, **kwargs):
+    data["e^vartheta"] = (
+        data["theta_PEST_r"][:, jnp.newaxis] * data["e^rho"]
+        + (1 + data["theta_PEST_t"])[:, jnp.newaxis] * data["e^theta"]
```

Comment:
This is incorrect; should remove the +1.  I have not checked the rest of the PR.